third_party/3/orjson.pyi

```diff
@@ -3,7 +3,7 @@ from typing import Any, Callable, Optional, Union
 __version__ = str
 
 def dumps(
-    __obj: Any,
+    obj: Any,
```

Comment:
Double underscore is how typeshed denotes positional-only arguments. This argument still appears to be positional-only, so this change should be reverted.

Reply:
Hmm, actually it seems `orjson.dumps`' signature just lies about several things, including this:
```
>>> import inspect
>>> import orjson
>>> inspect.signature(orjson.dumps)
<Signature (obj, /, default, option)>
>>> orjson.dumps(obj={"a": 2})
b'{"a":2}'
```
So I take it back, this seems fine.

Reply:
On the third hand, the pyi that orjson ships has it as positional-only https://github.com/ijl/orjson/blob/master/orjson.pyi

Reply:
I know. It seems like they need to change their documentation too. I will submit an issue on their repo too. Thanks!